### DISCUSSION TITLE
`brew install erlang` usage error on `erlc -h` with: File has no extension: /kernel

### DISCUSSION THREAD

**Discussion by esambo** [Homebrew/discussions#3840] [Everyday usage] [answered]
I'm on Apple M1 Pro using macOS Ventura 13.0 with Command Line Tools 14.1 RC2
No matter how I install erlang (erlang --build-from-source, erlang, erlang@24, erlang@23, erlang@22), I keep getting the following error:
`cd /` (the current directory is reflected in the error message)
`erlc -h`
```
File has no extension: /kernel
```

**Comment by esambo** [comment, marked as answer]
It looks like the problem is caused by the presence of the env. var. that for some reason doesn't work on macOS Ventura anymore:
`env | grep kernel`
```
ERL_AFLAGS=kernel shell_history enabled
```

Not sure why...

**Reply by gromgit** [reply]
Doesn't sound right, for two reasons:
1. The env var you list is for `erl`, but you're invoking `erlc`, which [docs doesn't list that var as being directly supported](https://www.erlang.org/doc/man/erlc.html).
2. That combo of `ERL_AFLAGS` and `erlc` doesn't work on any platform I've tried, not even Linux.

Are you sure it _ever_ worked for you?

**Reply by esambo** [reply]
`erlc -h` works just fine (without causing the error) if I use the correct syntax of:
```
export ERL_AFLAGS="-kernel shell_history enabled"
```

FYI, I used `erlc` after having googled around for that error message and found it mentioned in: https://helpmanual.io/help/erlc/
